user_guide/context.rst

```diff
@@ -139,19 +139,34 @@ your code more structured by allowing you to use multiple contexts in a single t
 suite.
 
 In order to customise the list of contexts your test suite requires, you need
-to fine-tune the suite configuration inside ``behat.yml``:
+to fine-tune the suite configuration inside ``behat.php``:
 
-.. code-block:: yaml
-
-    # behat.yml
+.. code-block:: php
 
-    default:
-        suites:
-            default:
-                contexts:
-                    - FeatureContext
-                    - SecondContext
-                    - ThirdContext
+    <?php
+    // behat.php
+
+    use App\Tests\Behat\Context\FeatureContext;
+    use App\Tests\Behat\Context\SecondContext;
+    use App\Tests\Behat\Context\ThirdContext;
```

Comment:
We should keep the contexts in all these examples in the root namespace, because "out of the box" just following the instructions on this page, these won't be autoloadable by behat (unless you put them in `features/bootstrap/App/Tests/Behat/Context/FeatureContext`).

Of course I realise that the vast majority of people these days are familiar with how to configure autoloading themselves e.g. through composer, but while we're providing / documenting Behat's custom autoloader I think the docs examples should be consistent with how that behaves.

```suggestion
    use FeatureContext;
    use SecondContext;
    use ThirdContext;
```